src/ukify/test/test_ukify.py

```diff
@@ -59,6 +59,8 @@
     slow_tests = True
 
 arg_tools = ['--tools', build_root] if build_root else []
+if pathlib.Path(f"{build_root}/linux{ukify.guess_efi_arch()}.efi.stub").exists():
```

Comment:
Maybe better to conditionalize with `build_root` before use?
```python
if build_root and pathlib.Path(...).exists():
```
Or,
```python
arg_tools = []
if build_root:
    arg_tools =+ ['--tools', build_root]
    if pathlib.Path(...).exists():
        arg_tools =+ [...]
```

Reply:
Why? It's the same result?

Reply:
Is it safe to use build_root in pathlib.Path() without checking?